operator/clusterutilities.go

```diff
@@ -392,6 +392,16 @@ func GetTablespaceNamePVCMap(clusterName string, tablespaceStorageTypeMap map[st
 	return tablespacePVCMap
 }
 
+// GetImagePrefix checks whether the imagePrefix value provided from the pgcluster
+// CRD has been set. If so, this value is returned, otherwise the default global value
+// is used. This supports both the CCPImagePrefix and the PGOImagePrefix
+func GetImagePrefix(crdImagePrefix, globalImagePrefix string) string {
+	if crdImagePrefix != "" {
+		return crdImagePrefix
+	}
+	return globalImagePrefix
+}
+
```

Comment:
Slightly picking a nit here, but wouldn't it be more appropriate to call it 'default' over 'global' since the value is being provided to the function?  Global, IMV, means that it's a static value that is accessible  'globally' and doesn't need to be passed as a parameter for such a function.

If it is truly 'global', then perhaps the following would be more appropriate?

```
func GetImagePrefix(imagePrefix string) string {
    if imagePrefix != "" {
        return imagePrefix
    }
    return GlobalimagePrefix
}
```

Otherwise, what you have but with...

```
func GetImagePrefix(imagePrefix string, defaultPrefix string) {
    ...
    return defaultPrefix
}
```

Just a thought.

Reply:
I agree with Adam on the first one.

Reply:
Understood, it is not a global value from an implementation perspective. I agree defaultPrefix is less confusing. I've updated it as such, thanks!

Reply:
So, giving this one more thought, it almost seems like this could/should be a `util` like function? It kind of seems like something that could/would be useful elsewhere perhaps? So why not make it available as such?

```
package util

func GetValueOrDefault(value string, default string) string {
    if value != "" {
        return value
    }

    return default
}
```

Thoughts?

Reply:
I think it makes sense, it looks like it results in less new package imports, and `util` is a bit more logical place at this point. I'll implement this change, thanks.